aframe/src/index.js

```diff
@@ -51,27 +51,28 @@ AFRAME.registerComponent('zesty-ad', {
         component.removeChild(component.lastChild);
       }
     }
-
-    if (!!component.getAttribute('visible') && !component.firstChild) {
-      this.init();
-    }
+    
+    // if (!!component.getAttribute('visible') && !component.firstChild) {
```

Comment:
Commenting this out for now as having it in causes the ad to be loaded multiple times...